components/Slider/src/MDCSlider.m

```diff
@@ -332,6 +333,14 @@ - (void)setHapticsEnabled:(BOOL)hapticsEnabled {
   }
 }
 
+- (void)setFullHapticsEnabled:(BOOL)fullHapticsEnabled {
+  if (@available(iOS 10.0, *)) {
+    if (_thumbTrack.numDiscreteValues >= 2) {
```

Comment:
what happens if the number of discrete values is 1?

Reply:
"if numDiscreteValues is set to one, then the thumb track will act as if numDiscreteValues is zero and will judge you silently." - in MDCThumbTrack.h
I think if the number of discrete values is 1 then everything is treated as a continuous slider because a slider would need at least two discrete values.

Reply:
awesome, thanks!